packages/terra-functional-testing/src/config/wdio.conf.js

```diff
@@ -129,7 +130,19 @@ exports.config = {
   // your test setup with almost no effort. Unlike plugins, they don't add new
   // commands. Instead, they hook themselves up into the test process.
   services: [
-    [TerraService],
+    [TerraService, {
+      /* Terra defined viewport sizes. */
+      terraViewports: {
+        tiny: { width: 470, height: 768 },
+        small: { width: 622, height: 768 },
+        medium: { width: 838, height: 768 },
+        large: { width: 1000, height: 768 },
+        huge: { width: 1300, height: 768 },
+        enormous: { width: 1500, height: 768 },
+      },
+      /* Use to change the form factor (test viewport) used in the wdio run. */
+      ...FORM_FACTOR && { formFactor: FORM_FACTOR },
```

Comment:
Should we update this to be huge by default?

Reply:
Updated: https://github.com/cerner/terra-toolkit/pull/438/commits/4d600076e3df0e9c99b327e4e0be28289810fa0a